docs/bidders/pubmatic.md

```diff
@@ -0,0 +1,33 @@
+# PubMatic Bidder
+
+## Test Request
+
+The following test parameters can be used to verify that Prebid Server is working properly with the 
+PubMatic adapter. This example includes an `imp` object with an PubMatic test publisher ID, ad slot,
+and sizes that would match with the test creative.
+
```

Comment:
Are we ok with only mentioning this example and two out of the four parameters that the `imp.ext` object can come with? What about a video `imp` example? Will that be left for another PR?